docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -628,3 +633,84 @@ The API returns the following response:
 // TESTRESPONSE[s/"end_time_in_millis": 1593094752019/"end_time_in_millis": $body.snapshots.1.end_time_in_millis/]
 // TESTRESPONSE[s/"duration_in_millis": 0/"duration_in_millis": $body.snapshots.0.duration_in_millis/]
 // TESTRESPONSE[s/"duration_in_millis": 1/"duration_in_millis": $body.snapshots.1.duration_in_millis/]
+
+
+The following request returns information for all snapshots that come after `snapshot_2` when sorted by snapshot name in the default
```

Comment:
I suspect that sorting after a given start/end time will be the most common use case. Maybe it's worth showing as an example?

Reply:
I'd love to but I don't see a way of doing this with a test because the timestamp on the request would have to be dynamic right?

Reply:
> the timestamp on the request would have to be dynamic right?

Or low enough to always return some snapshots in the response. But that's just a suggestion, let's ignore if that's too complicated or ugly.

Reply:
Fair point :) Added a docs run for this with a low timestamp now.